vulcanai2/tests/models/test_layers.py

```diff
@@ -70,6 +91,27 @@ def convunit(self):
             dropout=0.5
         )
 
+    @pytest.fixture
+    def test_convunit_parameters(self):
+        """Create a dictionary with parameters that 
+        aren't ConvUnit parameters"""
+        return dict(
+                    in_channels=10,
+                    out_features=10,
+                    initializer=nn.init.xavier_uniform_,
+                    bias_init=0.1,
+                    norm='batch',
+                    activation=nn.ReLU(),
+                    dropout=0.5
+                )
+    
+    def test_create_convunit(self, test_convunit_parameters):
+        """Check if passing wrong parameters raises TypeError"""
+        with pytest.raises(TypeError) as e:
+            DenseUnit(**test_convunit_parameters)
+            assert e.message == "__init__() got an unexpected \
```

Comment:
same comment about moving the assertion outside the with statement

Reply:
👍